core/consensus/aura/src/lib.rs

```diff
@@ -66,18 +66,6 @@ pub use consensus_common::SyncOracle;
 type AuthorityId<P> = <P as Pair>::Public;
 type Signature<P> = <P as Pair>::Signature;
 
-/// A handle to the network. This is generally implemented by providing some
-/// handle to a gossip service or similar.
-///
-/// Intended to be a lightweight handle such as an `Arc`.
-pub trait Network: Clone {
```

Comment:
even though it's dead, it's technically part of the public API so we should just deprecate for now.

Reply:
Got it.